### PR TITLE
updating httpoison

### PR DESCRIPTION
**Pull request by InoMurko**
debuggins issues that appeared in sentry

## Overview

Updating http client dependecy (httpoison bumped to 1.6.2), catching messages that cause exceptions and analysing them.
## Testing

debug

### CODE REVIEW

**Comment by InoMurko** [comment]
Screen Shot 2020-05-27 at 18 19 08
appears from time to time, don't know from where so I'm trying to analyse it

**Comment by InoMurko** [comment]
also part of https://github.com/omisego/elixir-omg/issues/1541

**Comment by boolafish** [comment]
closes #1529

File: apps/omg_eth/lib/omg_eth/ethereum_height_monitor.ex [hunk 1]
```diff
@@ -90,6 +90,23 @@ defmodule OMG.Eth.EthereumHeightMonitor do
     {:noreply, state}
   end
 
+  # we need to figure out where :sslsocket is getting closed
+  # because it randomly kills this process and block getter
+  # https://github.com/omisego/elixir-omg/issues/1541
```
Comment (boolafish) [inline apps/omg_eth/lib/omg_eth/ethereum_height_monitor.ex:95]: slightly encourage to link this one instead: https://github.com/omisego/elixir-omg/issues/1529

File: apps/omg_eth/lib/omg_eth/ethereum_height_monitor.ex [hunk 2]
```diff
@@ -90,6 +90,23 @@ defmodule OMG.Eth.EthereumHeightMonitor do
...
+  # https://sentry.io/organizations/omisego/issues/1582175010/?project=1477673
+  def handle_info({:ssl_closed, {:sslsocket, {:gen_tcp, port, :tls_connection, :undefined}, pids}}, state) do
```
Comment (boolafish) [inline apps/omg_eth/lib/omg_eth/ethereum_height_monitor.ex:97]: This is smart!! Never think of this. Cool elixir trick

**Comment by boolafish** [comment]
There is new occurrence on your new status get cache module

```
GenServer OMG.Watcher.API.StatusCache terminating
** (FunctionClauseError) no function clause matching in OMG.Watcher.API.StatusCache.handle_info/2
    (omg_watcher 0.4.8) lib/omg_watcher/api/status_cache.ex:60: OMG.Watcher.API.StatusCache.handle_info({:ssl_closed, {:sslsocket, {:gen_tcp, #Port<0.23>, :tls_connection, :undefined}, [#PID<0.3174.0>, #PID<0.3173.0>]}}, %OMG.Watcher.API.StatusCache{ets: :status_cache, integration_module: OMG.Watcher.API.StatusCache.External})
    (stdlib 3.12) gen_server.erl:637: :gen_server.try_dispatch/4
    (stdlib 3.12) gen_server.erl:711: :gen_server.handle_msg/6
    (stdlib 3.12) proc_lib.erl:249: :proc_lib.init_p_do_apply/3
Last message: {:ssl_closed, {:sslsocket, {:gen_tcp, #Port<0.23>, :tls_connection, :undefined}, [#PID<0.3174.0>, #PID<0.3173.0>]}}
```

might be interested to add log in that module too within this PR?

**Pull request merged**